core/cowdata.h

```diff
@@ -31,6 +31,8 @@
 #ifndef COWDATA_H_
 #define COWDATA_H_
 
+#include <string.h>
```

Comment:
What do we need `string.h` for here?

Reply:
memcpy()

Reply:
Ah right, I expected to be in a more low-level header, but it's indeed in `cstring`.